src/crimson/os/seastore/omap_manager/btree/btree_omap_manager.cc

```diff
@@ -106,6 +106,22 @@ BtreeOMapManager::omap_get_value(
   });
 }
 
+BtreeOMapManager::omap_set_keys_ret
+BtreeOMapManager::omap_set_keys(
+  omap_root_t &omap_root,
+  Transaction &t,
+  std::map<std::string, ceph::bufferlist>&& keys)
+{
+  return seastar::do_with(std::move(keys), [&, this](auto& keys) {
+    return crimson::do_for_each(
```

Comment:
nit, can we do this in parallel ?

Reply:
Um, does the current btree implementation support concurrent update? If so, I think there's no problem to do this in parallel.

Reply:
Um, does the current btree implementation support concurrent update? If so, I think there's no problem to do this in parallel.

Reply:
i think it does not matter if the btree implementation support concurrent update or not. `omap_set_key()` is a high level interface, so i think we should do this in parallel unless the order of setting the omap entries matters.

@athanatos @liu-chunmei  what do you think?

Reply:
@athanatos I remember I report some parallel issue. But I forgot if I found it in lba btree or other places.

Reply:
Just do it sequentially for now.